### ISSUE TITLE
Textarea in internet explorer

### ISSUE THREAD

**Issue by eliran2** [formly-js/ng-formly#544]
**I'm submitting a ...**  (check one with "x")
```
[X] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request
```

**Current behavior**
When trying to use type textarea in internet explorer it's raise an error.

**Expected behavior**

**Minimal reproduction of the problem with instructions**
Use type textarea and try to open in internet explorer.

**What is the motivation / use case for changing the behavior?**

**Please tell us about your environment:**

* **Angular version:** 2.0.X

* **Browser:** IE XX 
 
* **Language:**TypeScript

* **Node (for AoT issues):** `node --version` =

**Comment by aitboudad** [comment]
@eliran2 using `material` or `bootstrap` ?

**Comment by eliran2** [comment]
bootstrap

**Comment by aitboudad** [comment]
Just tested on IE-11 and everything work as expected see https://angular-5ioeet.stackblitz.io/, 
please ensure to include all required [polyfills](https://github.com/formly-js/ng-formly/blob/master/demo/src/polyfills.ts#L21-L35) for IE.

**Comment by aitboudad** [comment]
seems an error is raised `ERROR Error: Invalid property value.`, I'll look into it soon.

**Comment by eliran2** [comment]
run np install formly latest?

**Comment by aitboudad** [comment]
@eliran2 the fix is to set valid value for `cols` and `rows` attrs (IE doesn't allow undefined value  for attr), just seting them in the config will solve you issue:
```ts
type: 'textarea',
templateOptions: {
  cols: 1,
  rows: 1,
},
```
The fix will be included in `2.0.0-beta.1`, if you didn't yet switched into the new package name `@ngx-formly/bootstrap` see https://github.com/formly-js/ng-formly/issues/501#issuecomment-337271517

**Comment by eliran2** [comment]
Thank you very much (:
